### ISSUE TITLE
Tempus: Need a test for SolutionState::swapSolutionData()

### ISSUE THREAD

**Issue by ccober6** [trilinos/Trilinos#3330]
This has been tested by EMPIRE usage, but we do not have a Tempus test covering this.

@trilinos/tempus 

## Your Environment

- **Relevant repo SHA1s:**  
- **Relevant configure flags or configure script:**  
- **Operating system and version:**  
- **Compiler and TPL versions:**  

## Related Issues

* Blocks 
* Is blocked by 
* Follows 
* Precedes 
* Related to 
* Part of 
* Composed of

**Comment by ccober6** [comment]
No longer needed.
